public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
- [OPTIONAL] To make your codebase cleaner I recommend deleting all the boilerplates created by the CRA (texts, images, styles) from the `public `folder.